### PR TITLE
RFC: Add the ability to retrieve the raw camera data

### PR DESCRIPTION
**Pull request by RealOrangeOne**
This is the form of a 2d numpy array

## Checklist

- [ ] I have updated the relevant documentation
- [ ] I have added the `semver-major`, `semver-minor` or `semver-patch` label as appropriate
- [x] I would like multiple reviewers to approve my code before merging

## Why do you want to make these changes?

For uses outside zoloto, it's useful to read the raw pixel data from the camera without any additional processing.

## Which parts of the codebase do your changes affect?

Basically everything.

## How do your changes affect student or volunteer experience?

Moar data = moar gud

## Are your changes related to any existing issues or PRs? How so?

N/A

### CODE REVIEW

**Review by trickeydan — APPROVED** [review]
LGTM. Not sure it changes "everything" like your PR claims though.

CI is failing on linting, bit I'm happy for this to be merged once that is fixed.

File: stubs/numpy.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+from typing import Any
+
+# HACK: Ignore numpy types, for now. Requires py3.8 https://numpy.org/devdocs/reference/typing.html.
+ndarray = Any  # type: ignore
```
Comment (trickeydan) [inline stubs/numpy.py:4]: No

**Review by trickeydan — CHANGES_REQUESTED** [review]

File: stubs/numpy.py (hunk 1 above)
Comment (RealOrangeOne) [reply stubs/numpy.py:4]: Yes
